bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,228 @@
+            }
+        :param always_semicolon:
+            If True, adds a semicolon at the end.
+
+            Example:
+            .a {
+                color: tomato
+            }
+
+            After using the bear:
+            .a {
+                color: tomato;
+            }
+        :param color_case:
+            If set to 'upper', makes all colors with upper-case letters.
+            If set to 'lower', makes all colors with lower-case letters.
+
+            Example(for upper case):
+            .a {
+                color: #fff;
+            }
+
+            After using the bear:
+            .a {
+                color: #FFF;
+            }
+        :param block_indent:
+            If set to "\t", indents by a tab.
+
+            Example:
+            .a {
+            color: tomato;
+            }
+
+            After using the bear:
+            .a {
+                color: tomato;
+            }
+        :param color_shorthand:
+            If True, changes the hexa colors into short form.
+
+            Example(for True case):
+            .a {
+                color: #ffcc00;
+            }
+
+            After using the bear:
+            .a {
+                color: #fc0;
+            }
+        :param eof_newline:
+            If True, adds a newline at EOF.
+        :param leading_zero:
+            If False, deletes the zeroes preceding the dots.
+            If True, adds the zeroes preceding the dots.
+
+            Example(False case):
+            .a {
+                padding: 0.1vh;
+                font-size: 0.5em;
+            }
+
+            After using the bear:
+            .a {
+                padding: .1vh;
+                font-size: .5em;
+            }
+        :param quotes:
+            Choose between double quotes and single quotes.
+            If "single" - single quotes.
+            If "double" - double quotes.
+        :param sort_order_fallback:
+            If True, sorts the properties alphabetically.
+        :param space_before_colon:
+            If  " ", add a space before colon.
+            If "", remove a space before colon.
+        :param space_after_colon:
+            If " ", adds a space after colon.
+            If "", remove a space after colon.
+        :param space_between_declaration:
+            If "\n", adds a line break between declarations.
+        :param space_before_o_brace:
+            If True, adds a line break before opening brace.
+        :param space_after_o_brace:
+            If True, adds a line break after opening brace.
+        :param space_after_s_delim:
+            If "\n", adds a line break after selector delimiter
+
+            Example:
+            .a, .b{
+                color: tomato;
+            }
+
+            After using the bear:
+            .a,
+             .b{
+                color: tomato;
+             }
+        :param space_before_s_delim:
+            If " ", adds a space before selector delimiter.
```

Comment:
please change these that are like " " and have only 2 optoins, to be True/False (add a space or dont add a space. nobody will give the answer as " ")